src/Microsoft.Sbom.Api/SBOMValidator.cs

```diff
@@ -23,16 +26,25 @@ public class SbomValidator : ISBOMValidator
     private readonly IWorkflow<SbomParserBasedValidationWorkflow> sbomParserBasedValidationWorkflow;
     private readonly IRecorder recorder;
     private readonly IEnumerable<ConfigValidator> configValidators;
+    private readonly IConfiguration configuration;
+    private readonly ISbomConfigProvider sbomConfigs;
+    private readonly IFileSystemUtils fileSystemUtils;
 
     public SbomValidator(
         IWorkflow<SbomParserBasedValidationWorkflow> sbomParserBasedValidationWorkflow,
         IRecorder recorder,
         IEnumerable<ConfigValidator> configValidators,
-        ConfigSanitizer configSanitizer)
+        ConfigSanitizer configSanitizer,
```

Comment:
We can also remove this configSanitizer. Looks like I accidentally left this in. But it doesn't actually get used anywhere.